### PR TITLE
Update docs and appdata (for flatpak)

### PR DESCRIPTION
**Pull request by Laserlicht**
GOG is now supported

### CODE REVIEW

File: docs/INSTALL.md [hunk 1]
```diff
@@ -198,8 +198,15 @@ flatpak run io.github.ihhub.Fheroes2
 
 Alternatively, you can use an application manager like Discover, which is also available on Steam Deck.
 
-You have to copy the subdirectories `ANIM`, `DATA`, `MAPS` and `MUSIC` from the original game or demo directory to the
-`~/.var/app/io.github.ihhub.Fheroes2/data/fheroes2` directory. You will be asked when you start the application.
+After the first start you will be asked for the original files. There are three possibilities:
+1. Manual install
+2. Install GOG version (recommended)
+3. Install demo
```
Comment (ihhub) [inline docs/INSTALL.md:204]: If `Install GOG version (recommended)` is recommended should we just put it at first place?
```
1. Install GOG version (recommended)
2. Manual installation
3. Install demo
```

File: docs/INSTALL.md [hunk 2]
```diff
@@ -198,8 +198,15 @@ flatpak run io.github.ihhub.Fheroes2
...
+
+For the first option you have to copy the subdirectories `ANIM`, `DATA`, `MAPS` and `MUSIC` from the original game or demo directory to the
```
Comment (ihhub) [inline docs/INSTALL.md:206]: I suggest to rephrase it into `for the manual installation ...`. What do you think?

File: script/packaging/common/fheroes2.appdata.xml [hunk 3]
```diff
@@ -1,21 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-  <id>fheroes2.desktop</id>
+  <id>io.github.ihhub.Fheroes2</id>
```
Comment (ihhub) [inline script/packaging/common/fheroes2.appdata.xml:3]: I suggest to make `Fheroes2` as `fheroes2` as it is the correct abbreviation.

File: script/packaging/common/fheroes2.appdata.xml [hunk 4]
```diff
@@ -1,21 +1,36 @@
...
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0</project_license>
   <name>fheroes2</name>
   <developer_name>fheroes2 Resurrection team</developer_name>
-  <summary>fheroes2 is a recreation of Heroes of Might and Magic II game engine.</summary>
+  <summary>fheroes2 is a recreation of Heroes of Might and Magic II game engine</summary>
```
Comment (ihhub) [inline script/packaging/common/fheroes2.appdata.xml:8]: What if the reason we are removing the full stop at the end of the sentence?

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Laserlicht , I left multiple comments here. Could you please take a look?

File: docs/INSTALL.md (hunk 1 above)
Comment (Laserlicht) [reply docs/INSTALL.md]: Good point

File: docs/INSTALL.md (hunk 2 above)
Comment (Laserlicht) [reply docs/INSTALL.md]: Good idea :)

File: script/packaging/common/fheroes2.appdata.xml (hunk 3 above)
Comment (Laserlicht) [reply script/packaging/common/fheroes2.appdata.xml:3]: The ID is what was recommended to me to comply with the naming convention. So this is intentional.

File: script/packaging/common/fheroes2.appdata.xml (hunk 4 above)
Comment (Laserlicht) [reply script/packaging/common/fheroes2.appdata.xml:8]: I ran a validator over it and a dot at the end of the summary was displayed as a warning.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Laserlicht , a bunch of thanks for these corrections in the documentation!

**Pull request merged**